src/test/java/com/intershop/version/semantic/AssertVersion.java

```diff
@@ -0,0 +1,63 @@
+package com.intershop.version.semantic;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.Comparator;
+
+/**
+ * Assertions for compare versions.
+ * 
+ * @author <a href="mailto:hboutemy@apache.org">Hervé Boutemy</a>
+ */
+public class AssertVersion {
```

Comment:
Think about to implement a Hamcrest-matcher instead. Same purpose but well known among developers so it would be easier to maintain.